### DISCUSSION TITLE
Make the launched modes the only moment that the stations of the looping coaster have the RCT1 look

### DISCUSSION THREAD

**Discussion by TheClaudeQC** [OpenRCT2/OpenRCT2#23488] [Feature requests]
I kind of miss the "Chainlift" style of the rct2 vanilla looping coasters station pieces... and it would be cool if the 2 could coexist instead of only having the rct1 stations

![Parc sans nom 2024-12-27 14-05-45](https://github.com/user-attachments/assets/b55b50e0-cd74-4324-aef8-8fc5b8b33d5c)
A looping coaster station in OpenRCT2

![SCR2](https://github.com/user-attachments/assets/9d473370-22b7-4fcf-93c3-6d63508eb9d9)
A looping coaster station in RCT2 Vanilla

**Comment by SpartanFrederic104** [comment]
I'm guessing this could be implemented in a similar manner to the LSM Launched Roller Coaster where the station changes from using a chainlift to drive tires when using any powered launch mode (including the block sectioned version that would otherwise be unavailable to the Looping Roller Coaster)?

**Comment by ZeeMaji** [comment]
I believe it was mentioned semi-recently that this can be implemented with track style objects in the future. You could have one for the RCT1 look which is used when importing S4 as well as any old .parks and use the RCT2 look when importing S6 as well as presumably making that the default track style for the looping coaster in new .parks?

That would be much more accurate to the original games, so it seems like the best solution to me.
